cnames_active.js

```diff
@@ -3539,6 +3539,7 @@ var cnames_active = {
   "yu": "yuxizhe.github.io/yu",
   "yuko": "notmarx.github.io/Yuko-Website",
   "yup-phone": "abhisekp.github.io/yup-phone",
+  "yurbajs": "yurba.js.org",
```

Comment:
I'm not sure what the intent is here? You're requesting that `yurbajs.js.org` is pointed at `yurba.js.org`?

I suspect you want to point this as Vercel?

```suggestion
  "yurba": "cname.vercel-dns.com", // noCF
```